gulpfile.js

```diff
@@ -24,6 +35,8 @@ var sort            = require( 'gulp-sort' );
 var uglify          = require( 'gulp-uglify' );
 var wpPot           = require( 'gulp-wp-pot' );
 var zip             = require( 'gulp-zip' );
+var browserSync 	= require( 'browser-sync' ).create();
```

Comment:
The equals signs should ideally be aligned with the ones above it. This has been engrained in us as this is something the PHP linter would flag if this were a PHP file.

Reply:
👾